spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin2/sender/ZipkinRestTemplateSenderConfiguration.java

```diff
@@ -34,30 +34,46 @@
 import org.springframework.cloud.sleuth.zipkin2.ZipkinLoadBalancer;
 import org.springframework.cloud.sleuth.zipkin2.ZipkinProperties;
 import org.springframework.cloud.sleuth.zipkin2.ZipkinRestTemplateCustomizer;
+import org.springframework.cloud.sleuth.zipkin2.sender.http.BraveHttpInterceptorRemover;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Conditional;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.http.HttpMethod;
+import org.springframework.http.client.ClientHttpRequestInterceptor;
 import org.springframework.web.client.RequestCallback;
 import org.springframework.web.client.ResponseExtractor;
 import org.springframework.web.client.RestClientException;
 import org.springframework.web.client.RestTemplate;
 
+import zipkin2.reporter.Sender;
+
 @Configuration
 @ConditionalOnMissingBean(name = ZipkinAutoConfiguration.SENDER_BEAN_NAME)
 @Conditional(ZipkinSenderCondition.class)
 @EnableConfigurationProperties(ZipkinSenderProperties.class)
 class ZipkinRestTemplateSenderConfiguration {
 
+	/**
+	 * Zipkin rest template bean name. Name of the bean matters for supporting multiple tracing
+	 * systems.
+	 */
+	public static final String REST_TEMPLATE_BEAN_NAME = "zipkinRestTemplate";
```

Comment:
I think your intent is that this is public so you should move it to `ZipkinAutoConfiguration` with the other constants like this

Reply:
done